CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+# Contributing to Garden Linux
+
+## Code of conduct
+
+All members of the Garden Linux community must abide by the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md). Only by respecting each other can we develop a productive, collaborative community. Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting gardenlinux.opensource@sap.com and/or a Garden Linux project maintainer.
+
+## Contributing
+
+Garden Linux uses GitHub to manage reviews of pull requests.
+
+If you are a new contributor see: [Steps to Contribute](#Steps-to-Contribute)
+
+If you have a trivial fix or improvement, go ahead and create a pull request, addressing (with @...) a suitable maintainer of this repository (see CODEOWNERS of the repository you want to contribute to) in the description of the pull request.
+
+If you plan to do something more involved, first discuss your ideas on our mailing list gardenlinux@googlegroups.com. This will avoid unnecessary work and surely give you and us a good deal of inspiration.
```

Comment:
Is that mailing list in use? If so, https://groups.google.com/g/gardenlinux is a better link maybe and/or should be added so people can find out how to join/subscribe

Reply:
I will add the link at the bottom. The mailing list has just been created.

Reply:
Best to just link to that section of the document then instead of handing out the mailng list link - people might not read the whole document (yeah, I know...)